utilities/cassandra/cassandra_format_test.cc

```diff
@@ -122,6 +122,15 @@ TEST(ExpiringColumnTest, ExpiringColumn) {
       == 0);
 }
 
+TEST(TombstoneTest, TombstoneCollection) {
+  std::string hex = "7fffffff80000000000000000100599f4bc300055786e9d6e9eb";
```

Comment:
Can you add comments what does this mean? I assume this is carefully constructed.

Reply:
It's not a very clear test. I will rewrite it.